spec/memo_wise_spec.rb

```diff
@@ -276,19 +274,45 @@ def unmemoized_method
       expect(instance2.no_args_counter).to eq(0)
     end
 
-    it "keeps private methods private" do
-      expect(instance.private_methods.include?(:private_memowise_method)).
-        to eq(true)
+    context "with private methods" do
+      it "keeps private methods private" do
+        expect(instance.private_methods.include?(:private_memowise_method)).
```

Comment:
This might be clearer as

```ruby
expect(instance.private_methods).to include(: private_memowise_method)
```

(And same comment below.) Thoughts?